src/components/About.tsx

```diff
@@ -0,0 +1,26 @@
+
+import AboutPanel from 'components/home/homeScreen/about'
+import HamburgerMenu from './home/common/hamburgerMenu/hamburgerMenu';
+import RightSection from './common/RightSection';
+
+const About = () => {
+  return (
```

Comment:
You can implicitly return here:
```suggestion
const About = () => (
```